src/explorer/js/app.js

```diff
@@ -174,6 +174,24 @@
               }
             };
 
+            var overwrite = false; 
+            for (var i = 0; i < explorer.fileManager.files().length; i++) {              
+              for (var k = 0; k < (current.children().length - 1); k++) {
+                if (current.children()[k].name == explorer.fileManager.files()[i].name) {
+                  var choseOverwrite = window.confirm("Duplicate file names detected." + 
+                                                      " Click OK to overwrite and replace all duplicates" +
+                                                      " or CANCEL to proceed without overwriting.");
+                  if (choseOverwrite) {
+                    overwrite = true;
+                  }
+                  break; 
+                }
+              } 
+              if (overwrite == true) {
```

Comment:
I think it's fine to break if the user has made a choice, rather than repeatedly prompting them, regardless of if they chose overwrite to be true/false. Perhaps we can include another flag above that we check here to break?